spec/classes/sensu_package_spec.rb

```diff
@@ -320,7 +320,32 @@
           it { should_not contain_yumrepo('sensu') }
           it { should contain_package('sensu').with( :require => nil ) }
         end
+      end
+
+      context 'Darwin' do
+        let(:facts) do
+          {
+            :osfamily => 'Darwin',
+            :kernel   => 'Darwin',
+            :macosx_productversion_major => '10.12',
+          }
+        end
 
+        context 'default' do
+        directories.each do |dir|
```

Comment:
need a test to show that the contents of `/etc/default/sensu` is `EMBEDDED_RUBY=true`

https://github.com/sensu/sensu-puppet/pull/863/files#diff-1576180ba8964fff8d837b3256127ebcR206

Reply:
added here 6575759f24f1864ebab6ec240f1f70a8738f3cb8

Reply:
added missing tests in 6575759f24f1864ebab6ec240f1f70a8738f3cb8